lyonjs/shortvid.io
# feat(app): generate button

## 🤔 Why do you want to make those changes?

Now that we have a server to generate video, we need to add the control in the front to make it usable by the users

## 🧑‍🔬 How did you make them?

I created a component that is used in the form and that allow the user to generate then download the video

## 🧪 How to check them?

- check the code
- check the preview

The generation on the preview should fail because of cors management so here is a screenshot video of the us :


https://github.com/lyonjs/shortvid.io/assets/72607059/9795515a-b7bc-4916-b46e-4192ee37e1a7
